src/components/Articles/SingleArticle.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+import axiosInstance from "../utilities/axios"
+
+const SingleArticle = () => {
+  const { articleId } = useParams()
+  const [article, setArticle] = useState(null)
+
+  useEffect(() => {
+    const fetchArticle = async () => {
+      try {
+        const response = await axiosInstance.get(`/articles/${articleId}`)
```

Comment:
Ideally you should separate out your api calls into a separate api.js function and file 
https://notes.northcoders.com/courses/js-front-end/axios#using-axios-to-make-http-requests